src/screens/staking/lib/staking_sdk/core.ts

```diff
@@ -51,13 +57,18 @@ export const mainNetworkDenom: Record<StakingNetworkId, CoinDenom | null> = {
   [StakingNetworkId.CosmosHub]: CoinDenom.ATOM,
   [StakingNetworkId.CosmosHubTestnet]: CoinDenom.ATOM,
   [StakingNetworkId.DyDx]: CoinDenom.DYDX,
+  [StakingNetworkId.Kava]: CoinDenom.KAVA,
+  [StakingNetworkId.KavaTestnet]: CoinDenom.KAVA,
   [StakingNetworkId.Osmosis]: CoinDenom.OSMO,
   [StakingNetworkId.Regen]: CoinDenom.REGEN,
+  [StakingNetworkId.Stargaze]: CoinDenom.STARS,
+  [StakingNetworkId.StargazeTestnet]: CoinDenom.STARS,
 };
 
 export const testnetNetworks = new Set([
   StakingNetworkId.CosmosHubTestnet,
   StakingNetworkId.CelestiaTestnet,
+  StakingNetworkId.KavaTestnet,
```

Comment:
I think have to add stargaze testnet here

Reply:
Yup, related to the missing push 🙈